docs/source/periodic_table.rst

```diff
@@ -0,0 +1,80 @@
+Periodic Table
+==============
+
+Full access to the periodic table is also available within QCPortal. This
```

Comment:
I don't care but is QCPortal what you meant?

Reply:
Nope. Make a patch?